src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -526,23 +530,46 @@ private RecommendedSymbols GetSymbolsOffOfExpressionInConstantPattern(Expression
             if (originalExpression is null)
                 return default;
 
-            var boundSymbol = _context.SemanticModel.GetSymbolInfo(originalExpression, _cancellationToken).Symbol;
+            var semanticModel = _context.SemanticModel;
+            var boundSymbol = semanticModel.GetSymbolInfo(originalExpression, _cancellationToken);
+
+            if (boundSymbol.Symbol is not INamespaceOrTypeSymbol namespaceOrType)
+            {
+                // Likely a Color Color case, so we reinterpret the bound symbol into a type
+                if (originalExpression is IdentifierNameSyntax identifier)
+                {
+                    var reinterpretedBinding = semanticModel.GetSpeculativeSymbolInfo(identifier.SpanStart, identifier, SpeculativeBindingOption.BindAsTypeOrNamespace);
+                    var reinterpretedSymbol = reinterpretedBinding.GetAnySymbol();
+
+                    // The reinterpretation must be a type
+                    if (reinterpretedSymbol is not INamedTypeSymbol reinterprettedNamedType)
+                        return default;
+
+                    var expression = originalExpression.WalkDownParentheses();
+
+                    return GetSymbolsOffOfBoundExpressionWorker(reinterpretedBinding, originalExpression, expression, reinterprettedNamedType, false, false);
```

Comment:
named parameters for 'false'.
wrap args.

Reply:
Done